components/form/FAutocompleteInput.vue

```diff
@@ -301,6 +308,25 @@ function formatOption(option: FMenuOption) {
   );
 }
 
+function handleMenuToggle(toggleValue: boolean) {
+  if (toggleValue) {
+    // When opening the menu
+    matchInputWithSelectedOption();
+  } else {
+    // When closing the menu
+    setTimeout(matchInputWithSelectedOption, 15); // Wait for Popper to properly close the menu before changing input value, to avoid seeing menu content breaks due to menu items changing
```

Comment:
I supposed you tested it, but a `nextTick` does not the job I guess ?

Reply:
Indeed, it does not suffice : in fact even 10 `nextTicks` weren't enough 😅